tests/testdata/enckey.enc

```diff
@@ -1 +1 @@
-95c74e8a65bf9575370c07f175c9a88e:776ca294b312cb3f44b94481b3b010759912b34e2485e9f457d160b21760f7d50109fcfb4a9c0783464410a5894299e59bbf3a7ae44019bed54420a9c4ba18da5cc78dbc19bec42884831d7c3fad86695f3c629ffc41b3d7da22c53f23cf1f4b76a178ebb9636c4eb4bc454ad1e3cd47becbfd923f479b85642c93a5aa8ea2e85595fca2f45c1e0361d92748e8ccc8590f57f3d9849d7ac022183ade71263974b051574c54a636080a9bbf5bce0a09745812056ea8945666916a9019faf48ee89e5f0f8a2520ac044254524335ac3155
+95c74e8a65bf9575370c07f175c9a88e:776ca294b312cb3f44b94481b3b010759912b34e2485e9f457d160b21760f7d50109fcfb4a9c0783464410a5894299e59bbf3a7ae44019bed54420a9c4ba18da5cc78dbc19bec42884831d7c3fad86695f3c629ffc41b3d7da22c53f23cf1f4b76a178ebb9636c4eb4bc454ad1e3cd47becbfd923f479b85642c93a5aa8ea2e85595fca2f45c1e0361d92748e8ccc8590f57f3d9849d7ac022183ade71263974b051574c54a636080a9bbf5bce0a09745812056ea8945666916a9019faf48ee8fe859253e15be6404094e4f32f3db9b4
```

Comment:
Oups, why has the key changed?
THis should not happen - what happens to existing installations?

Reply:
When i am finished with the versioning of the encryption, this should be fixed.

Reply:
OK, cool.
Can you please leave the old value as commented out, so that we know the "target-value".